pkg/ocm/ocm.go

```diff
@@ -66,6 +74,23 @@ func (client ocmClient) GetAWSAccountClaim(clusterID string) (*awsv1alpha1.Accou
 	return ac, err
 }
 
+// GetClusterInfo returns cluster information from ocm by using either internal, external id or the cluster name 
+// Returns a v1.Cluster object or an error
+func (client ocmClient) GetClusterInfo(identifier string) (*v1.Cluster, error) {
+	q := fmt.Sprintf("(id like '%[1]s' or external_id like '%[1]s' or display_name like '%[1]s')", identifier)
+	resp, err := client.conn.ClustersMgmt().V1().Clusters().List().Search(q).Send()
```

Comment:
I think i've seen this code like 3 times already. not in this codebase but altogether.. might suggest adding an `ocm-sdk-go` util for this

Reply:
yes we are going to encapsulate the ocm communication for our tests in an own interface and struct the next PR, to make it also testable